README.md

```diff
@@ -49,41 +49,41 @@ python3 -m pip install PyTweet
 
 ## Usage
 
-Before using PyTweet you have to setup an application in [here](https://apps.twitter.com), for more comfortable experience you can create an application inside a project. Most endpoints require the client to have read, post and direct_messages app permissions and elevated access type. For more accessibility you can create a dev environment to support events and other premium endpoints. If you have any questions, please open an issue or ask in the official [discord server](https://discord.gg/nxZCE9EbVr)
+Before using PyTweet you have to setup an application [here](https://apps.twitter.com). For a more comfortable experience, you can create an application inside a project. Most endpoints require the client to have read, post and direct_messages app permissions and elevated access type. For more accessibility you can create a dev environment to support events and other premium endpoints. If you have any questions, please open an issue or ask in the official [PyTweet Discord](https://discord.gg/nxZCE9EbVr).
```

Comment:
Do you mind changing "post" to write? I forgot to change it.

Reply:
Ok.